### DISCUSSION TITLE
Lite design question: Why do we need a run() method

### DISCUSSION THREAD

**Discussion by aazzolini** [Lightning-AI/pytorch-lightning#10206] [Idea pool]
It looks completely out of place to me and misleading. I really like the simplicitly of LightningLite as a _concrete_ class.
The run method here is completely out of place. The docstring is wrong, because nothing gets accelerated because it is inside the "run" function, no other places call the "run" method.

I believe it would be great if we could remove the "run" method from here and make LightingLite a concrete class. 

https://github.com/PyTorchLightning/pytorch-lightning/blob/c614cf0c3d7a252bfd01ff8516f9f42f1c1b055a/pytorch_lightning/lite/lite.py#L143

**Comment by justusschock** [comment]
This is a new experimental API to be released with 1.5. It works without the trainer but does not support all the convenient features the lightning Trainer does, just bare hardware acceleration.

> The docstring is wrong, because nothing gets accelerated because it is inside the "run" function, no other places call the "run" method.

No, it is correct because if you look at the examples for this, the `run` method is supposed to be implemented and called directly by the user.
